docs/components/expression-metadata.js

```diff
@@ -199,6 +199,7 @@ for (const name in CompoundExpression.definitions) {
 }
 
 delete types['error'];
+delete types['accumulated']; // skip documenting `accumulated` since it is internal use only
```

Comment:
Should this still be deleted, given that it is now usable as part of a custom reduce expression?

Reply:
Right, just readded it.